src/Core/Entity/Contact.php

```diff
@@ -28,7 +28,7 @@ class Contact extends StaticEntity
 	protected $created;
 	protected $updated;
 	protected $can_notify;
-
+	public $country_code; // we only want this for validation, needs to be unset before saving
```

Comment:
for reviewer: ideas to avoid adding this  property outside of state/definition since I only need it for the endpoint and validation of number and won't be saved

Reply:
There is a pattern of using vars like this in other entities so I think it's ok but will look for another pattern.